ffi/capi/src/properties_enums.rs

```diff
@@ -72,10 +75,57 @@ pub mod ffi {
     }
 
     impl BidiClass {
-        pub fn to_integer(self) -> u8 {
+        #[diplomat::rust_link(icu::properties::props::EnumeratedProperty::for_char, FnInTrait)]
+        #[cfg(feature = "compiled_data")]
+        pub fn for_char(ch: DiplomatChar) -> Self {
+            icu_properties::CodePointMapData::<props::BidiClass>::new()
+                .get32(ch)
+                .into()
+        }
+
+        #[diplomat::rust_link(icu::properties::PropertyNamesLongBorrowed::get, FnInStruct)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesLong, Struct, hidden)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesLongBorrowed, Struct, hidden)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesLong::new, FnInStruct, hidden)]
+        #[diplomat::rust_link(icu::properties::PropertyNamesLongBorrowed::new, FnInStruct, hidden)]
+        #[diplomat::rust_link(
+            icu::properties::props::NamedEnumeratedProperty::long_name,
+            FnInTrait,
+            hidden
+        )]
+        #[cfg(feature = "compiled_data")]
+        /// Get the "long" name of this property value (returns empty if property value is unknown)
```

Comment:
I don't think this is great UX. It should return `Option<&'static str>`, so that in Dart it is `String?`.

Reply:
Oh, right, we can do that